package.json

```diff
@@ -25,14 +25,17 @@
     "start:db": "cd ./typescript/db && yarn start",
     "stop:db": "cd ./typescript/db && yarn stop",
     "bundlewatch:web": "cd ./typescript/web && yarn build && yarn bundlewatch",
-    "codegen": "graphql-codegen --config ./codegen.yml",
+    "codegen": "yarn codegen:data && yarn codegen:web",
     "codegen:all": "yarn codegen",
     "codegen:ci": "graphql-codegen --config ./codegen.yml && git diff --exit-code",
+    "codegen:data": "graphql-codegen --config ./codegen.yml",
+    "codegen:web": "cd typescript/web && yarn codegen",
     "storybook": "start-storybook -s ./typescript/web/public -p 6006",
     "storybook:all": "start-storybook -s ./typescript/web/public -p 6006",
     "build-storybook": "build-storybook",
     "chromatic": "npx chromatic",
-    "release": "cd ./typescript/web && yarn release"
+    "release": "cd ./typescript/web && yarn release",
+    "generate": "yarn codegen && yarn generate:apollo"
```

Comment:
Do we still need this script?

Reply:
In case someone wants to generate the types without running the web server ?

Reply:
But `generate:apollo` does not even seems to exist...

`yarn generate` was however called in `.github/workflows/ci.yml but it's always for the `db`.

Maybe we could just rename the `generate` into `codegen` and harmonize things and use a `codegen:db` script and integrate it like what you've done for the `data` and `web` `codegen` scripts?

Reply:
No you were right, this command is not used, we replace it when we harmonize the `codegen` commands earlier, I will delete it